setup.py

```diff
@@ -8,6 +8,8 @@
 import ah_bootstrap
 from setuptools import setup
 
+ah_bootstrap._check_submodule("asdf-standard")
+
```

Comment:
:-1: Let's please not rely on non-public functions.  Automatically initializing submodules is certainly convenient, but please wait until something like the discussion in astropy/astropy-helpers#74 is available.  In the meantime not having this is hardly a showstopper.  You can set up an alias like:
```
clone-all = clone --recursive
```

and use that to initialize and update all submodules when cloning a repository.